analyticsexample/build.gradle

```diff
@@ -3,31 +3,27 @@ apply plugin: 'kotlin-android'
 apply plugin: 'kotlin-android-extensions'
 
 android {
-    compileSdkVersion 27
+    compileSdkVersion project.compileSdkVersion
 
     defaultConfig {
         applicationId "com.bitmovin.exoplayeranalyticsexample"
-        minSdkVersion 16
-        targetSdkVersion 27
-        versionCode 1
-        versionName "1.0"
-        testInstrumentationRunner "android.support.test.runner.AndroidJUnitRunner"
+        minSdkVersion project.minSdkVersion
+        targetSdkVersion project.targetSdkVersion
+        versionCode project.versionCode
```

Comment:
Not sure if it's intended to have the collector version in the examples as well, we have never updated them before. 
@Tigraine ?

Reply:
Shouldn't really matter but why not.. Makes it cleaner to debug?

Reply:
as we do not release these testing/developing apps (`bitmovinanalyticsexample` and `analyticsexample`) it should not matter, before they had always version `1.0`, now we have version in all submodules consistent.